src/ngraph/frontend/onnx_import/op/scatter_nd.cpp

```diff
@@ -0,0 +1,46 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <memory>
+
+#include "ngraph/op/fused/scatter_nd.hpp"
+#include "utils/common.hpp"
```

Comment:
I think you don't need `#include "utils/common.hpp"` but you should include your `scatter_nd.hpp` file here.

Reply:
Done